google/api_core/retry_streaming.py

```diff
@@ -0,0 +1,221 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Generator wrapper for retryable streaming RPCs.
+This function will be used when initilizing a retry with
+``Retry(is_stream=True)``.
+
+When ``is_stream=False``, the target is treated as a callable,
```

Comment:
Yeah, my only worry is it would be adding a lot of text to the existing wall of text in `Retry`. I thought it would be better to give a brief description and say "for more information, see retry_target_stream" instead.

Do you think this should be moved into the main Retry docstring?

Reply:
Yes, I think that's the right level of abstraction for these comments. I suggest keeping the more immediate, more likely-to-be-referenced parts at the top of the comment, and these details further down

Reply:
done